lib/compare.js

```diff
@@ -51,8 +50,7 @@ const strictEqualArray = (arr1, arr2) => {
   for (let i = 0; i < len1; i++) {
     const item1 = arr1[i];
     const item2 = arr2[i];
-    if (item1 === item2) continue;
```

Comment:
The same question as for the `equalObject` function above, `strictEqual` already includes this check.